netdata.spec.in

```diff
@@ -1,90 +1,100 @@
-BuildRoot:	%{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+BuildRoot:	%{tmpdir}/%{name}-%{version}-root-%(id -u -n)
 
 %description
-Real-time performance monitoring, in the greatest possible detail!
+netdata is the fastest way to visualize metrics. It is a resource
+efficient, highly optimized system for collecting and visualizing any
+type of realtime timeseries data, from CPU usage, disk activity, SQL
+queries, API calls, web site visitors, etc.
+
+netdata tries to visualize the truth of now, in its greatest detail,
+so that you can get insights of what is happening now and what just
+happened, on your systems and applications.
+
+%prep
+%setup -q
+
+%build
+%{__aclocal} -I m4
+%{__autoconf}
+%{__autoheader}
+%{__automake}
+%configure \
+	--docdir=%{_docdir}/%{name}-%{version} \
+	--with-zlib \
+	--with-math \
+	%{?with_nfacct:--enable-plugin-nfacct} \
+	--with-user=netdata
+%{__make} %{?_smp_mflags}
+
+%install
+rm -rf $RPM_BUILD_ROOT
+%{__make} %{?_smp_mflags} install \
+	DESTDIR=$RPM_BUILD_ROOT
+
+%{__rm} $RPM_BUILD_ROOT/var/{cache,log}/netdata/.keep
+
+%if %{with systemd}
+install -d $RPM_BUILD_ROOT%{_unitdir}
+cp -p system/netdata.service $RPM_BUILD_ROOT%{_unitdir}/netdata.service
+%endif
 
 %pre
 getent group netdata > /dev/null || groupadd -r netdata
 getent passwd netdata > /dev/null || useradd -r -g netdata -c netdata -s /sbin/nologin -d / netdata
 
-%if 0%{do_systemd}
+%if %{with systemd}
 %post
 %systemd_post netdata.service
+
 %preun
 %systemd_preun netdata.service
+
 %postun
 %systemd_postun_with_restart netdata.service
 %endif
 
-%global ovirt_create_user_engine \
-%_ovirt_create_user %{engine_user} %{engine_uid} %{engine_group} %{engine_gid} "%{ovirt_user_description}" %{engine_state}
-
-
-%prep
-%setup -q -n %{name}-@PACKAGE_VERSION@
-
-%build
-%configure \
-	--docdir="%{_docdir}/%{name}-%{version}" \
-	--with-zlib \
-	--with-math \
-	--with-user=netdata \
-%if !0%{?rhel}
-	--enable-plugin-nfacct \
-%endif
-	%{?conf}
-make %{?_smp_mflags}
-
-%install
-rm -rf "%{buildroot}"
-make %{?_smp_mflags} install DESTDIR="%{buildroot}"
-find "%{buildroot}" -name .keep -exec rm {} \;
-
-%if 0%{do_systemd}
-install -m 0755 -d "%{buildroot}/%{_unitdir}"
-install -m 0644 system/netdata.service "%{buildroot}/%{_unitdir}"
-%endif
+%clean
+rm -rf $RPM_BUILD_ROOT
 
 %files
-%attr(-, netdata, netdata) %dir %{_localstatedir}/cache/%{name}/
-%attr(-, netdata, netdata) %dir %{_localstatedir}/log/%{name}/
-%config(noreplace) %{_sysconfdir}/%{name}/
-%{_datadir}/%{name}/
-%{_libexecdir}/%{name}/
-%{_sbindir}/%{name}
-
-%if 0%{do_systemd}
-%{_unitdir}/netdata.service
-%endif
+%dir %{_sysconfdir}/%{name}
+%config(noreplace) %verify(not md5 mtime size) %{_sysconfdir}/%{name}/*.conf
+%attr(755,root,root) %{_sbindir}/%{name}
+%{_datadir}/%{name}
+%{_libexecdir}/%{name}
+%{?with_systemd:%{_unitdir}/netdata.service}
+%attr(755,netdata,netdata) %dir %{_localstatedir}/cache/%{name}
+%attr(755,netdata,netdata) %dir %{_localstatedir}/log/%{name}
```

Comment:
On 06.04.2016 00:00, Alon Bar-Lev wrote:
> again... rpmbuild should be executed on mock or a sane umask (if not 
> run by rpmbuild at start).

pff.

[~/rpm/packages/netdata/netdata (rpmspec)↑⚡] ➔ git push
fatal: remote error:
   GitHub is offline for maintenance. See http://status.github.com for 
more info.

it was the last push i needed to make.

so will continue tomorrow!

-- 
glen